solutions/silver/usaco-923.mdx

```diff
@@ -52,14 +78,15 @@ $200\times200$ matrix):
 ### Using Kadane's
 
 Now, we've reduced this problem to finding the max submatrix sum on a matrix,
-given that we can have two disjoint submatrices.
-Though it's a tall talk, let's try and tackle this bit by bit, first by considering
-if there's only *1* matrix we can paint.
+given that we can have two disjoint submatrices. Though it's a tall talk, let's
```

Comment:
```suggestion
given that we can have two disjoint submatrices. Though it's a tall task, let's
```

Reply:
my brother in christ you wrote the solution

Reply:
yeah i make typos lol